backend/accounts/account.go

```diff
@@ -40,7 +40,8 @@ type Interface interface {
 	Name() string
 	// Initialize only starts the initialization, the account is not initialized right afterwards.
```

Comment:
does the "initialization" also means "synchronization" here? if so, I'd add it to the comment:

> // Initialize starts account sync and returns immediately without waiting for completion

Reply:
Good catch. Changed the words.